dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -339,8 +339,10 @@ def _ProcessThread(self, client):
     collected_flow_data = self._DownloadFiles(client, flow_id)
     if collected_flow_data:
       print('{0!s}: Downloaded: {1:s}'.format(flow_id, collected_flow_data))
-      fqdn = client.data.os_info.fqdn.lower()
-      self.state.output.append((fqdn, collected_flow_data))
+      self.state.StoreContainer(containers.File(
```

Comment:
Please separate the creation of the container from storing it, to make this easier to read and debug

Reply:
I'm going to stop mentioning this now, as this seems like a pattern, and I think this should change everywhere.